ocamltest/ocamltest.org

```diff
@@ -327,13 +327,15 @@ required steps to achieve this:
 1. We slightly modify the test block in =hello.ml=, as follows:
    #+begin_src
    (* TEST
-   flags = "-w +33"
+   {
+     flags = "-w +33";
+   }
```

Comment:
It looks like the test blocks in `ocamltest.org` are still using the mandatory-outer-brace syntax, and could benefit from an update.

Reply:
You're right. Done.